src/modules/logicModule/interfaces/ILM_PC_Staking_v1.sol

```diff
@@ -98,25 +98,25 @@ interface ILM_PC_Staking_v1 {
     //--------------------------------------------------------------------------
     // Mutating Functions
 
-    /// @notice Stake a specified amount of tokens to earn rewards
-    /// @dev should tokens already be staked, then the sending address will collect the rewards up until this point
-    /// @dev Fee on transfer tokens are currently not supported
-    /// @param amount : how much token should be staked
+    /// @notice Stake a specified amount of tokens to earn rewards.
+    /// @dev	should tokens already be staked, then the sending address will collect the rewards up until this point.
```

Comment:
Might also benefit from a capitalized first word, not sure

Reply:
Yeah actually a lot weren't capitalized. Tried to catch as many as possible.